frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -660,7 +660,7 @@ function ReaderFooter:resetLayout(force_reset)
     else
         bar_height = self.settings.progress_style_thick_height or PROGRESS_BAR_STYLE_THICK_DEFAULT_HEIGHT
     end
-    self.progress_bar.height = Screen:scaleBySize(bar_height)
+    self.progress_bar:setHeight(bar_height)
```

Comment:
There's another instance of this @ L1836

(Which I'm not quite sure is actually necessary at all, because it has nothing whatsoever to do with what that function's supposed to do...)

Reply:
I thought I had searched for other instances, obviously I missed at least one.